cmd/podman/containers/ps.go

```diff
@@ -98,14 +98,6 @@ func checkFlags(c *cobra.Command) error {
 	if listOpts.Last >= 0 && listOpts.Latest {
 		return errors.Errorf("last and latest are mutually exclusive")
 	}
-	// Filter on status forces all
```

Comment:
It's not immediately clear what's replacing this - do we still ensure that all is set?

Reply:
Yes it is checked later in the code path again.
https://github.com/containers/podman/blob/a6f7afd41729edbe0d8cbf21bab5b475c6a13f9e/pkg/ps/ps.go#L40-L44

Reply:
Wow. Wonder how that slipped in. OK, thanks!